__tests__/app.test.js

```diff
@@ -5,6 +5,7 @@ const app = require("../app");
 const db = require("../db/connection");
 const seed = require("../db/seeds/seed");
 const data = require("../db/data/test-data");
+require("jest-sorted");
```

Comment:
You shouldn't need to require the library in here - you just need to make sure you add in the setup code in oyur package.json as the jest sorted docs suggest!